### PR TITLE
ign to gz

### PR DESCRIPTION
**Pull request by ahcorde**
# 🦟 Bug fix

## Summary
Replace ign with gz

## Checklist
- [ ] Signed all commits for DCO
- [ ] Added tests
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if the library has them)
- [ ] `codecheck` passed (See [contributing](https://gazebosim.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://gazebosim.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Agazebosim+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: ros_gz_bridge/README.md [hunk 1]
```diff
@@ -232,7 +232,7 @@ On terminal A, start the service bridge:
 
 On terminal B, start Gazebo, it will be paused by default:
 
-`ign gazebo shapes.sdf`
+`gz gazebo shapes.sdf`
```
Comment (azeey) [inline ros_gz_bridge/README.md]:
```suggestion
`gz sim shapes.sdf`
```

File: ros_gz_bridge/README.md [hunk 2]
```diff
@@ -148,14 +148,14 @@ First we start Gazebo Sim (don't forget to hit play, or Gazebo Sim won't generat
 
 ```
 # Shell A:
-ign gazebo sensors_demo.sdf
+gz gazebo sensors_demo.sdf
```
Comment (azeey) [inline ros_gz_bridge/README.md]:
```suggestion
gz sim sensors_demo.sdf
```

**Review by azeey — APPROVED** [review]

**Comment by ahcorde** [comment]
This is documentation, I will merge it

File: README_RENAME.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ This allows users to do either of these and get equivalent behavior:
 
 ```bash
 ros2 run ros_gz parameter_bridge [...]
-ros2 run ros_ign parameter_bridge [...]  # Will emit deprecation warning
+ros2 run ros_gz parameter_bridge [...]  # Will emit deprecation warning
```
Comment (moriarty) [inline README_RENAME.md:10]: maybe this got caught by a find replace?

**Pull request merged**